tests/ui/pages/desktop/home.py

```diff
@@ -39,3 +52,120 @@ def users(self):
                 """Number of users that have downloaded the extension"""
                 users_str = self.find_element(*self._users_locator).text
                 return int(users_str.split()[0].replace(',', ''))
+
+    class FeaturedExtensions(Region):
+        """ Featured Extension region """
+
+        _root_locator = (By.ID, 'featured-extensions')
+
+        _extension_locator = (By.CSS_SELECTOR, 'section > li > div')
+        _see_all_locator = (By.CSS_SELECTOR, 'h2 > a')
+
+        @property
+        def extensions(self):
+            return [self.Extension(self.page, el) for el in self.find_elements(
+                *self._extension_locator)]
+
+        class Extension(Region):
+
+            _root_locator = (By.CLASS_NAME, 'addon-hovercard')
```

Comment:
No need for `_root_locator` then the object is passed an element on construction.